modules/apps/saved-content/saved-content-service/src/main/java/com/liferay/saved/content/service/impl/SavedContentEntryLocalServiceImpl.java

```diff
@@ -19,4 +29,64 @@
 )
 public class SavedContentEntryLocalServiceImpl
 	extends SavedContentEntryLocalServiceBaseImpl {
+
+	@Override
+	public SavedContentEntry addSavedContentEntry(
+			long groupId, long userId, String className, long classPK)
+		throws PortalException {
+
+		_validate(
+			CompanyThreadLocal.getCompanyId(), userId, className, classPK);
+
+		long savedContentEntryId = counterLocalService.increment(
+			SavedContentEntry.class.getName());
+
+		SavedContentEntry savedContentEntry =
+			savedContentEntryPersistence.create(savedContentEntryId);
+
+		savedContentEntry.setGroupId(groupId);
+
+		User user = _userLocalService.getUserById(userId);
+
+		savedContentEntry.setUserId(user.getUserId());
+		savedContentEntry.setUserName(user.getFullName());
+
+		savedContentEntry.setCreateDate(new Date());
+		savedContentEntry.setModifiedDate(new Date());
```

Comment:
Very minor thing: wouldn't it be safer to reuse the same `Date` instance here? When adding an entry, the create and modified date should be identical. Creating two different date instances could give two dates with very tiny differences.

Reply:
good point.